force-app/main/default/staticresources/act_postcalltask.resource

```diff
@@ -59,7 +59,13 @@ limitations under the License.
 
         contact.onConnected(function(contactOnConnected) {
           connect.getLog().info("ACToolkit:PostCallTask:onAgentHandler:ContactOnConnectedHandler invoked");
-          if (!contactOnConnected.isInbound()) {
+          /*
+           * Due to how callback works need to add a check so that it triggers
+           * the startActiveCall() method. Callbacks are always inbound but hit
+           * here when connecting to the customer so need to add a check to see
+           */
+          var type = contactOnConnected.getType();
```

Comment:
The variable type is only used in one place so there isn't a reason to create a variable for it.

The next line could simply be `if (!contactOnConnected.isInbound() || contactOnConnected.getType() === 'queue_callback')`